### PR TITLE
Added Comment to ActorMarketRole

### PR DESCRIPTION
**Pull request by mimse**

### CODE REVIEW

File: source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql [hunk 1]
```diff
@@ -0,0 +1,3 @@
+ALTER TABLE [dbo].[MarketRole]
+    ADD [Comment] nvarchar(max) NULL DEFAULT('')
```
Comment (xnetsrak) [inline source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql:2]: Why not keep it with Null value as Default, when it is nullable ?

**Review by xnetsrak — COMMENTED** [review]
Looks fine, only 2 questions .
See comment.
Do we need a test with Comment = null ?

File: source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql (hunk 1 above)
Comment (mimse) [reply source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql:2]: It was a copy / paste of another DB change. And I though it wouldnt hurt to have an empty string instead of null. But it could as well be null.
What do you think?

Comment (mimse) [reply source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql:2]: We know the database allow the value to be null, so I dont know how much value those test would provide?

Comment (xnetsrak) [reply source/Energinet.DataHub.MarketParticipant.ApplyDBMigrationsApp/Scripts/B-001/Model/202211041100 Add Comment to MarketRole.sql:2]: Ok

**Review by xnetsrak — APPROVED** [review]
